docs/reference/filters.md

```diff
@@ -4,8 +4,9 @@ The parameters can be strings, regex or float64 / int
 
 * `string` is a string surrounded by double quotes (`"`)
 * `regex` is a regular expression, surrounded by `/`, e.g. `/^www\.example\.org(:\d+)?$/`
-* `int` / `float64` are usual (decimal) numbers like `401` or `1.23456`
-* `time` is a string in double quotes, parseable by [time.Duration](https://godoc.org/time#ParseDuration))
+* `int` an integer decimal number like `401` or `1.0`
+* `float64` a decimal number like `401` or `1.23456`
+* `duration` is a string in double quotes, parsable by [time.Duration](https://godoc.org/time#ParseDuration) or a decimal number of milliseconds
```

Comment:
It is nice to have single duration type, but we need to check that all implementations support decimal milliseconds. E.g. ratelimit family supports undocumented integer number of _seconds_

Reply:
See https://github.com/zalando/skipper/pull/1814#issuecomment-895053332